vnext/ReactWindowsCore/Modules/AppThemeModule.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "AppThemeModule.h"
+
+namespace react { namespace windows {
+
+//
+// AppTheme
+//
+
+AppTheme::AppTheme() = default;
+AppTheme::~AppTheme() = default;
+
+// TODO: real implementation
```

Comment:
>TODO: real implementation [](start = 3, length = 25)

IIUC if the OS is in dark theme when you launch the app, and the theme is never changed, the module will incorrectly report the theme as light, correct?  If so I'd like to this addressed asap.  Please log a bug, even better if you can fix as part of this checkin.

Reply:
It's overridden in AppThemeModuleUwp.cpp and does report the correct theme on launch.